server/main.lua

```diff
@@ -39,12 +39,37 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleOwner = GetVehicleOwner(plate)
+		local bolo, title, boloId = GetBoloStatus(plate, title, boloId)

```

Comment:
```suggestion
		local bolo, title, boloId = GetBoloStatus(plate)

```

The second and third arguments `title` and `boloId` are not needed, they are nil values. You can test this by putting a print statement above this function and logging title and boloId, you will see they dont exist at this point.